.github/workflows/change-file-in-pr.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     steps:
       - name: Checkout PR code
-        uses: actions/checkout@v3
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 #v3.6.0
```

Comment:
why not update to v4

Reply:
That will be in a follow up PR. first I a updating to the commit has. And then i will be adding dependabot to all repos to automatically update the actions. There is a prerequisite to use commit hashes first though